### ISSUE TITLE
Response validation not working as expected

### ISSUE THREAD

**Issue by guy-microsoft** [cdimascio/express-openapi-validator#836]
**Describe the bug**
When I allow only 200 response code with application/json content, any response code is allowed if I set the response body as a string in my code.

**To Reproduce**
`import * as express from "express";
import { Request, Response } from "express";

export const router = express.Router();
/**
 * @openapi
 * components:
 *   schemas:
 *     Person:
 *       type: object
 *       properties:
 *         name:
 *           type: string
 *         age:
 *           type: integer
 *           format: int32
 */

/**
 * @openapi
 * /v1/resourceX/{id}:
 *   get:
 *     description: Get Person by ID
 *     parameters:
 *       - in: path
 *         name: id
 *         required: true
 *         schema:
 *           type: boolean
 *
 *     responses:
 *       '200':
 *         description: The person.
 *         content:
 *           application/json:
 *             schema:
 *               $ref: '#/components/schemas/Person'
 */
router.get(
    '/:id',
    async (req: Request, res: Response) => {
            res.status(209).send();
    }
);`

**Actual behavior** 
Response passes the validation.

**Expected behavior**
I would expected the response validation to fail the response.

**Examples and context**
If I use res.status(209).json({}) instead of res.status(209).send(), validation works as expected

**Comment by always-akshat** [comment]
Hey @guy-microsoft , Did you ever figure this out?

**Comment by guy-microsoft** [comment]
No, but I only use .json (or .send with a json payload), so it mitigates that.
